spec/rubocop/cop/rspec/contain_exactly_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::ContainExactly do
+  it 'flags `contain_exactly` with only splat arguments' do
+    expect_offense(<<-RUBY)
+      it { is_expected.to contain_exactly(*array1, *array2) }
```

Comment:
Can we add more tests please?
contain_exactly(*[1,2,3])
contain_exactly(*a.merge(b))
contain_exactly(*(a + b))
Guess the later two are not handled, and that's OK for the initial version, but lets have a test for the basic case of a literal array
how about also contain_exactly(*%w(a b))

Reply:
Do you mean `contain_exactly(*%w(a b))`? Added